### PR TITLE
Added some more context to the CLI exclude/include/match options

### PR DESCRIPTION
**Pull request by alexander-zierhut**
Added some documentation to the CLI. As discussed in #624

### CODE REVIEW

File: cmd/minify/README.md [hunk 1]
```diff
@@ -208,6 +208,8 @@ The patterns for `--match`, `--include`, and `--exclude` can be either a glob or
 
 Match will filters all files by the given pattern, eg. `--match '*.css'` will only minify CSS files. The `--include` and `--exclude` options allow to add or remove certain files or directories and is interpreted in the order given. For example, `minify -rvo out/ --exclude 'src/*/**' --include 'src/foo/**' src/` will minify the directory `src/`, except for `src/*/...` where `*` is not `foo`.
 
+You may define multiple patterns within one option, such as: `--exclude '**/folder1/**' '**/folder2/**' '**/folder3/**'` Doing this might result in unexpected behaviour when it is followed immediately by the input files, as this would be interpreted as another pattern, and not as inputs. `--exclude dir_to_exclude folder_input` Instead format accordingly: `--exclude dir_to_exclude -- folder_input`. This may be subject to change.
```
Comment (pydsigner) [inline cmd/minify/README.md:211]: This does sound like helpful information to add. Is there basis to include the `This may be subject to change.`?

Comment (tdewolff) [reply cmd/minify/README.md:211]: Not really planning on changing it, unless someone has compelling arguments that the usage causes too much surprise I guess. I'll remove the line for now

**Comment by tdewolff** [comment]
Thanks for the PR!

File: cmd/minify/README.md (hunk 1 above)
Comment (alexander-zierhut) [reply cmd/minify/README.md:211]: @pydsigner 
The "subject to change" came from here: https://github.com/tdewolff/minify/issues/624#issuecomment-1785736134

>  I'm not 100% convinced if this is the way of "least surprises" by users...

Comment (alexander-zierhut) [reply cmd/minify/README.md:211]: Resolved in: 173ecfef60053f5bbff16b4e66e884488e883a44

**Pull request merged**